packages/math-input/src/components/__tests__/two-page-keypad_test.tsx

```diff
@@ -8,7 +8,6 @@ describe("<TwoPageKeyPage />", () => {
         // Arrange
         const wrapper = mount(
             <TwoPageKeypad
-                paginationEnabled={true}
```

Comment:
This is provided by Redux, so it's erroneous to have it as a prop

Reply:
They fail with and without the prop. https://khanacademy.atlassian.net/browse/LC-753